backend/package.json

```diff
@@ -9,7 +9,7 @@
     "start": "nodemon server.js",
     "pretest": "eslint actions/",
     "prestart": "cross-env ENV='dev' node ./script.js",
-    "predeploy": "cross-env ENV='prod' node ./script.js"
```

Comment:
Did you intend to remove this row?

Reply:
`,`

Reply:
Oh, I read only first commit about this part. The second commit makes it back.